api/envoy/extensions/stat_sinks/open_telemetry/v3/open_telemetry.proto

```diff
@@ -44,4 +44,10 @@ message SinkConfig {
   // If set to true, metric names will be represented as the tag extracted name instead
   // of the full metric name. Default value is true.
   google.protobuf.BoolValue use_tag_extracted_name = 5;
+
+  // If set, emitted stats names will be prepended with a prefix, so full stat name will be
+  // <prefix>.<stats_name>. For example, if the stat name is "foo.bar" and prefix is
+  // 'pre', the full stat name will be "pre.foo.bar". If this field is not set, there is no
```

Comment:
Nit: weird mix of quotes, not sure how this looks in the rendered RST. @phlax is there a better reference to style we should point folks at?